src/webportal/src/app/job-submission/components/tools/job-ssh.jsx

```diff
@@ -138,20 +154,33 @@ export const JobSSH = ({
           content={`Choose SSH public key for job. Users should maintain the SSH private key themselves.`}
         />
       </Stack>
-      <Hint>
-        Enable Users SSH to allow user attach job containers through
-        corresponding ssh private key.
-      </Hint>
       <Toggle
-        label='Enable Job SSH'
+        label={
+          <div>
+            Enable Job SSH{' '}
+            <TooltipHost content='Enable Job SSH to allow SSH between job containers through ssh <taskrole_name>-<taskrole_index>.'>
+              <Icon iconName='Info' aria-label='Info tooltip' />
+            </TooltipHost>
+          </div>
+        }
         inlineLabel={true}
         checked={sshPlugin.jobssh === true}
         onChange={(ev, isChecked) => {
           _onChangeExtras('jobssh', isChecked);
         }}
       />
       <Toggle
-        label='Enable User SSH'
+        label={
+          <div>
+            Enable User SSH{' '}
```

Comment:
better to use padding to align the space between label and icon

Reply:
Done